pkg/ebpf/mappings.go

```diff
@@ -78,6 +78,32 @@ func AddNameMappingsCollection(coll *ebpf.Collection, module string) {
 	})
 }
 
+// AddNameMappingsForMap adds the full name mappings for the provided ebpf map
+func AddNameMappingsForMap(m *ebpf.Map, module string) {
+	mappingLock.Lock()
+	defer mappingLock.Unlock()
+
+	if info, err := m.Info(); err == nil {
```

Comment:
The inner parts of these functions are the same as `iterateMaps` and `iterateProgs`. Could you extract a common function for both use cases?

Reply:
I am using the MapInfo and ProgramInfo to get the name in all cases now. That should be okay right?

Reply:
Okay passing the names directly now